vividus-plugin-lighthouse/src/main/resources/org/vividus/lighthouse/lighthouse.ftl

```diff
@@ -0,0 +1,283 @@
+<#noparse>
```

Comment:
what is the source?

Reply:
what is a source of the code from this FTL?

Reply:
added upgrade information